subsys/testsuite/include/zephyr/fff_extensions.h

```diff
@@ -0,0 +1,74 @@
+/*
+ * Copyright(c) 2023 Legrand North America, LLC.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+#ifndef ZEPHYR_SUBSYS_TESTSUITE_INCLUDE_ZEPHYR_FFF_EXTENSIONS_H_
+#define ZEPHYR_SUBSYS_TESTSUITE_INCLUDE_ZEPHYR_FFF_EXTENSIONS_H_
+
+#include <zephyr/fff.h>
+#include <zephyr/sys/util.h> /* for CONTAINER_OF */
+
+
+/*
```

Comment:
nit: add extra `*` here and lets create a doxygen group for this file. Once done, please link it to `doc/develop/test/ztest.rst` under `Mocking via FFF` so we can make this more discoverable for others.

Reply:
Updated as requested.